### ISSUE TITLE
Gene interaction perturbation network deciphers a high-resolution taxonomy in colorectal cancer

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#25]
## TL;DR

Molecular subtypes of colorectal cancer (CRC) are currently identified via the snapshot transcriptional profiles, largely ignoring the dynamic changes of gene expressions. Conversely, biological networks remain relatively stable irrespective of time and condition. Here, we introduce an individual-specific gene interaction perturbation network-based (GIN) approach and identify six GIN subtypes with distinguishing features.

### Paper Link

https://elifesciences.org/articles/81114

### Author/Institution

Zaoqu Liu (University, Zhengzhou, China)

## Overview

In this paper, the investigators investigate CRC tumor heterogeneity by using a clustering method to understand perturbation of gene networks. The approach is robust and resulted in identification of significant differences in gene expression signals. The investigators identified six distinct gene interaction networks (GINs) that characterize tumor landscapes with varying degrees of oncogenic driver mutations, immune infiltration, and drug susceptibilities. These results provide a solid contribution to the field that, if validated, could be utilized as a useful predictive and prognostic correlative biomarkers in future clinical trials.

## Methods

colorectal cancer. survival analysis. network analysis.

## Cite

Zaoqu Liu Siyuan Weng Qin Dang Hui Xu Yuqing Ren Chunguang Guo Zhe Xing Zhenqiang Sun Xinwei Han (2022) Gene interaction perturbation network deciphers a high-resolution taxonomy in colorectal cancer eLife 11:e81114.
https://doi.org/10.7554/eLife.81114
